README.md

```diff
@@ -17,13 +17,14 @@
 
 ## Contents
 
-* [Installation](#installation)
+* [Node.js Installation and Usage](#node.js-installation-and-usage)
    * [Example: Create a transaction](#example-create-a-transaction)
+* [Browser Installation and Usage](#browser-installation-and-usage)
 * [Documentation](#bigchaindb-documentation)
 * [Authors](#authors)
 * [License](#license)
 
-## Installation
+## Node.js Installation and Usage
```

Comment:
This is true also for browsers. I'd rename this section to something like `Installation and Usage with package managers (npm/yarn)`.